components/home/TssUpdates.tsx

```diff
@@ -0,0 +1,102 @@
+import { ListPanel } from "nhsuk-react-components";
+import { useAppSelector } from "../../redux/hooks/hooks";
+import { useMemo } from "react";
+import { Post } from "../../models/WPPost";
+
+type PostSummary = Pick<Post, "id" | "title" | "excerpt">;
+
+export const TssUpdates: React.FC = () => {
+  const whatsNewHeader = "What's New";
+  const whatsNewStatus = useAppSelector(state => state.tssUpdates.status);
+  const whatsNewPosts: Post[] = useAppSelector(
+    state => state.tssUpdates.tssUpdates
+  );
+
+  const filteredPosts = useMemo(() => {
+    const filteredPosts: PostSummary[] = [];
+    if (whatsNewPosts.length === 0) {
+      return [];
+    }
+    for (const post of whatsNewPosts) {
+      if (post.tags.includes(20)) {
+        filteredPosts.push({
+          id: post.id,
+          title: post.title,
+          excerpt: post.excerpt
+        });
+      }
+    }
+    return filteredPosts;
+  }, [whatsNewPosts]);
+
+  const addWhatsNewPosts = (): JSX.Element[] | JSX.Element => {
+    if (filteredPosts.length === 0) {
+      return <p data-cy="noUpdates">No updates available</p>;
+    }
+    return filteredPosts.map(post => {
+      return (
+        <ListPanel.Item key={post.id}>
+          <h3 data-cy={`postTitle${post.id}`}>{post.title.rendered}</h3>
+          <p data-cy={`postExcerpt${post.id}`}>
+            {extractTextFromHTML(post.excerpt.rendered)}
```

Comment:
Tested on post with no manual excerpt - wordpress generates an automatic one itself, so this is fine.